app/core/ViewLoadTimer.coffee

```diff
@@ -87,16 +91,57 @@ class ViewLoadTimer
         entries = performance.getEntriesByType('resource').filter((r) => _.string.startsWith(r.name, location.origin))
+        essentialEntries = _.filter(entries, (entry) =>
+          return false if /vimeo/.test(entry.name)
+          return false if /TestView/.test(entry.name)
+          return false if /file\/db\//.test(entry.name) # Don't count DB images
+          return false if /\/db\/analytics\.log\.event/.test(entry.name) # Don't count analytics events
+          return true if /\/db\//.test(entry.name) # Do count DB data
+          return true if /chunks\/.*bundle.js/.test(entry.name)
+          return true if /app.js/.test(entry.name)
+          return true if /boot.js/.test(entry.name)
+          return true if /ace/.test(entry.name)
+          return true if /esper/.test(entry.name)
+          return true if /aether/.test(entry.name)
+          return true if /esper.modern.js/.test(entry.name)
+          return true if /lodash.js/.test(entry.name)
+          return true if /run-tests.js/.test(entry.name)
+          return true if /setImmediate.js/.test(entry.name)
+          return true if /web-dev-listener.js/.test(entry.name)
+          return true if /world/.test(entry.name)
+          return true if /app.css/.test(entry.name)
+          return false
+        )
+        if VIEW_LOAD_LOG
+          _.each {
+            'All resources': entries
+            'Essential resources': essentialEntries
+          }, (entryList, groupLabel) ->
+            console.groupCollapsed groupLabel
+            console.log "(transferSize in bytes: file path)"
+            console.log "#{entry.transferSize}: #{entry.name}" for entry in entryList
+            console.groupEnd()
         totalEncodedBodySize = _.reduce(entries, ((total, entry) -> total + entry.encodedBodySize), 0)
+        totalEssentialEncodedBodySize = _.reduce(essentialEntries, ((total, entry) -> total + entry.encodedBodySize), 0)
         totalTransferSize = _.reduce(entries, ((total, entry) -> total + entry.transferSize), 0)
+        totalEssentialTransferSize = _.reduce(essentialEntries, ((total, entry) -> total + entry.transferSize), 0)
+        console.log "totalTransferSize: #{totalTransferSize}" if VIEW_LOAD_LOG
+        console.log "totalEssentialTransferSize: #{totalEssentialTransferSize}" if VIEW_LOAD_LOG
         cachedResources = _.size(_.filter(entries, (entry) -> entry.transferSize / entry.encodedBodySize < 0.1))
+        cachedEssentialResources = _.size(_.filter(essentialEntries, (entry) -> entry.transferSize / entry.encodedBodySize < 0.1))
         totalResources = _.size(entries)
-        resourceInfo = { totalEncodedBodySize, totalTransferSize, cachedResources, totalResources }
+        totalEssentialResources = _.size(essentialEntries)
+        resourceInfo = { totalEncodedBodySize, totalTransferSize, cachedResources, totalResources,
+          totalEssentialEncodedBodySize, totalEssentialTransferSize, cachedEssentialResources, totalEssentialResources }
```

Comment:
These props should be omitted from snowplow event, to make sure they fit the existing schema.